lib/libs/api/package/itemExists.ts

```diff
@@ -1,7 +1,7 @@
 import { BaseIndex } from "lib/packages/shared-types/opensearch";
 import { getDomain, getOsNamespace } from "libs/utils";
 
-import * as os from "../../../libs/opensearch-lib";
+import * as os from "../../opensearch-lib";
```

Comment:
import * as os from "libs/opensearch-lib"; is how we do it in every other file though I do not think we should say as os personally.  For whatever reason this one particular  import does it diferently

Reply:
Oh, this is the `[import/no-useless-path-segments](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-useless-path-segments.md)`. It's basically the same path, but simplified